.github/workflows/web-service.yml

```diff
@@ -4,26 +4,40 @@ on:
   workflow_dispatch:
     inputs:
       model_name:
+        description: Name of the model
         type: string
-        default: 'smbert'
-        description: Name of the model to use
+        default: xaynia
+        required: true
       model_version:
+        description: Version of the model
         type: string
-        default: 'v0003'
-        description: Version of the model to use
+        default: v0002
+        required: true
+      runtime_name:
+        description: Name of the runtime
+        type: string
+        default: ort
+        required: true
+      runtime_version:
+        description: Version of the runtime
+        type: string
+        default: v1.15.1
+        required: true
       platform:
         description: Platform to build the image for
         type: choice
-        default: 'arm64'
         options:
           - arm64
           - amd64
+        default: arm64
+        required: true
       tag:
-        type: string
         description: >
           A tag for image identification. This override the default.
-          Default is <branch>-<timestamp>-<commit hash>-<model_name>_<model_version>-<platform>.
+          Default is <branch>-<timestamp>-<commit hash>-<model_name>_<model_version>-<runtime_name>_<runtime_version>-<platform>.
```

Comment:
I'm not sure why we need this, the code can switch between runtime with a configuration options and the ort version can be take from the commit.

Reply:
The space for the tag is limited so I don't want to overload it with info that we can easily get somewhere else

Reply:
i removed it.